flutter-studio/src/io/flutter/FlutterStudioStartupActivity.java

```diff
@@ -37,17 +54,17 @@
   public void runActivity(@NotNull Project project) {
     MessageBusConnection connection = project.getMessageBus().connect(project);
     // GRADLE_SYNC_TOPIC is not public in Android Studio 3.5. It is in 3.6. It isn't defined in 3.4.
-    Topic topic = getStaticFieldValue(GradleSyncState.class, Topic.class, "GRADLE_SYNC_TOPIC");
-    assert topic != null;
     //noinspection unchecked
-    connection.subscribe((Topic<GradleSyncListener>)topic, makeSyncListener(project));
+    Topic<GradleSyncListener> topic = getStaticFieldValue(GradleSyncState.class, Topic.class, "GRADLE_SYNC_TOPIC");
```

Comment:
There's a lot of add to app code in `FlutterStudioStartupActivity`; perhaps split the code out into a separate class and reference it from here?